### PR TITLE
using ngOptions to populate marital status

### PR DESCRIPTION
**Pull request by IngageMaddoxT**

### CODE REVIEW

File: crossroads.net/core/components/volunteer_applications/kc_adult_application/kidsClubAdultApplication.controller.js [hunk 1]
```diff
@@ -26,6 +26,7 @@ var moment = require('moment');
     vm.format = 'MM/dd/yyyy';
     vm.gradeLevelSelected = gradeLevelSelected;
     vm.locationSelected = locationSelected;
+    vm.maritalStati = maritalStati();
```
Comment (danrye) [inline crossroads.net/core/components/volunteer_applications/kc_adult_application/kidsClubAdultApplication.controller.js]: @IngageMaddoxT nitpik... statuses (http://www.merriam-webster.com/dictionary/status)

File: crossroads.net/core/components/volunteer_applications/kc_adult_application/kidsClubAdultApplication.controller.js [hunk 2]
```diff
@@ -79,6 +80,16 @@ var moment = require('moment');
       return false;
     }
 
+    function maritalStati() {
```
Comment (danrye) [inline crossroads.net/core/components/volunteer_applications/kc_adult_application/kidsClubAdultApplication.controller.js]: See above

**Comment by danrye** [comment]
Why are these statuses being fetched from Ministry Platform?

**Comment by danrye** [comment]
You could use ProfileReferenceData, though I'm sure we'd do this differently now that we've learned so much.  https://github.com/crdschurch/crds-angular/blob/development/crossroads.net/app/profile/personal/profile_personal_controller.js

**Comment by IngageMaddoxT** [comment]
Performance call.  One less api call for something that's not likely to change.  And these forms are "not the long term plan", a.k.a Black Pulp

**Comment by danrye** [comment]
:+1:

**Pull request merged**